modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -263,6 +263,37 @@ will return a JSON object of the form:
 }
 ```
 
+#### 2.1.6 Single project dicoms tar files
+
+GET /projects/$ProjectName/dicoms
+
+will return a JSON object of the form:
+
+```js
+{
+  "Dicoms": [
+    {
+      "Candidate": "587630",
+      "PSCID": "DCC090",
+      "Entity_type": "Human",
+      "Visit": "V1",
+      "Visit_date": "2018-04-20",
+      "CenterID": "1",
+      "Site": "Data Coordinating Center",
+      "date_acquired": "2018-04-20",
+      "date_first_archived": "2019-05-23 13:46:39",
+      "date_last_archived": "2019-05-23 13:46:39",
+      "tarchiveid": "27",
+      "DicomArchiveID": "1.2.840.113745.101000.1022000.39911.6153.5242769",
+      "Archive": "2018/DCM_2018-04-20_ImagingUpload-13-42-zcoZR0.tar",
+      "Source": "/tmp/ImagingUpload-13-42-zcoZR0",
+      "FileName": "DCM_2018-04-20_ImagingUpload-13-42-zcoZR0.tar"
+    },
+    ...
+  ]
+}
+```
```

Comment:
There should be a mention about the since parameter.

Reply:
what is the since parameter?

Reply:
https://github.com/aces/Loris/pull/6775/files#diff-abcc8bdf59aa52372d7e9df59277bafe7a4479d355250919912ff4a087939113R135

Reply:
I'm confused by this thread.. it seems to be mentioned?

Reply:
I added something about the since parameter after xlecours comment

Reply:
thanks @spell00  - @xlecours could you take a look when you have a moment? this feels close to success :)